add-chain/testdata/monorepo/op-node/rollup_faultproofs.json

```diff
@@ -21,13 +21,13 @@
     "seq_window_size": 3600,
     "channel_timeout": 300,
     "l1_chain_id": 11155111,
-    "l2_chain_id": 11155421,
+    "l2_chain_id": 4206903,
     "regolith_time": 0,
     "canyon_time": 0,
     "delta_time": 0,
     "batch_inbox_address": "0xff00000000000000000000000000000011155421",
-    "deposit_contract_address": "0x6509f2a854ba7441039fce3b959d5badd2ffcfcd",
-    "l1_system_config_address": "0x4af802b3010e07845b2b8c2250126e9ac0bdb6b9",
+    "deposit_contract_address": "0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6",
+    "l1_system_config_address": "0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7",
```

Comment:
How did you choose these test data? Did you do a fresh L1 deployment?

Reply:
`l2_chain_id` needs to be unique compared to others in the superchain. Since this PR relies on the `add-chain` tests writing artifacts to the superchain package prior to `add-chain check-rollup-config` importing the superchain package and running its tests, I needed to change the mock data chain ids so that the test artifacts could all exist simultaneously.

Reply:
Within the op-node code, the following rollup config address fields are set prior to writing the `rollup.json` file:
```		
DepositContractAddress: common.Address(addrs.OptimismPortalProxy),
L1SystemConfigAddress:  common.Address(addrs.SystemConfigProxy),
```
In rollup_faultproofs.json those address values are:
```
    "deposit_contract_address": "0x6509f2a854ba7441039fce3b959d5badd2ffcfcd",
    "l1_system_config_address": "0x4af802b3010e07845b2b8c2250126e9ac0bdb6b9",
```
However in the faultproofs .deploy file the addresses are:
```
  "OptimismPortalProxy": "0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6",
  "SystemConfigProxy": "0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7",
```
I changed the values in rollup_faultproofs.json to match the ones in .deploy because I believe rollup_faultproofs.json was constructed in a faulty (pardon the pun) way that is not representative of a real user flow. So I believe this change fixes a bug in the mock test data